RecoTracker/FinalTrackSelectors/plugins/TrackCollectionFilterCloner.cc

```diff
@@ -81,9 +81,7 @@ void TrackCollectionFilterCloner::fillDescriptions(edm::ConfigurationDescription
   desc.add<edm::InputTag>("originalMVAVals", edm::InputTag());
   desc.add<edm::InputTag>("originalQualVals",edm::InputTag());
   desc.add<std::string>  ("minQuality",      "loose");
-  edm::ParameterSetDescription clonerDesc;
-  TrackCollectionCloner::fill(clonerDesc);
```

Comment:
This is a defect of the original code: the Cloner is constructed form the full config not the cloner paramSet...